pkg/doc/vc/profile/profile.go

```diff
@@ -45,23 +48,31 @@ type DataProfile struct {
 	OverwriteIssuer         bool                               `json:"overwriteIssuer"`
 }
 
+// HolderProfile struct for holder profile
+type HolderProfile struct {
+	Name                    string                             `json:"name"`
+	DID                     string                             `json:"did"`
+	SignatureType           string                             `json:"signatureType"`
```

Comment:
we should have a TODO to switch to underscore separation (as we did with sidetree).

Reply:
Created an issue https://github.com/trustbloc/edge-service/issues/326